neo/Ledger/Blockchain.cs

```diff
@@ -228,7 +228,7 @@ private void OnImport(IEnumerable<Block> blocks)
             foreach (Block block in blocks)
             {
                 if (block.Index <= Height) continue;
-                if (block.Index != Height + 1)
+                if (block.Index != Height + 1 || block.PrevHash != CurrentBlockHash)
                     throw new InvalidOperationException();
```

Comment:
What happens after the exception? Shouldn't we use some message inside this exception? Something like "Invalid block during import" or equivalent?

Reply:
Will do.